docker-compose.yml

```diff
@@ -21,6 +21,17 @@ services:
             - ./api-dummy/public:/srv/app/public
         depends_on:
             - api-dummy-fpm
+    api-gateway:
+        image: nginx:1.13
+        ports:
+            - '8085:80'
+        volumes:
+            - ./api-gateway/.docker/nginx-server.conf.template:/etc/nginx/conf.d/default.conf.template
+        command: sh -c "envsubst \"`env | awk -F = '{printf \" $$%s\", $$1}'`\" < /etc/nginx/conf.d/default.conf.template > /etc/nginx/conf.d/default.conf && cat /etc/nginx/conf.d/default.conf && nginx -g 'daemon off;'"
```

Comment:
magic but explicit way to specify the configuration; later on the whole configuration file can be mounted if there is an orchestrator